lib/stream_data.ex

```diff
@@ -1164,12 +1164,14 @@ defmodule StreamData do
   Shrinks towards smallest maps and towards shrinking keys and values according
```

Comment:
I need to update the docs here to reflect the api change

Reply:
@davydog187 no worries, I quickly pushed an updated to the docs (and fixed the other comment from Michal). :)